### ISSUE TITLE
Failed to call the revokeRefreshToken and revokeAccessToken methods

### ISSUE THREAD

**Issue by fengzhihenxs** [damienbod/angular-auth-oidc-client#873]
Hello, I failed when calling the revokeRefreshToken and revokeAccessToken methods, why?
![image](https://user-images.githubusercontent.com/37610470/95940121-3ec07d00-0e10-11eb-8b40-4043990acfce.png)

During debugging, I found that the requested URL when revokeRefreshToken was null, it should be caused by this, I don’t know how to set or get RevocationEndpointUrl
![image](https://user-images.githubusercontent.com/37610470/95940384-c27a6980-0e10-11eb-9363-c1f153b4fc02.png)

**Comment by damienbod** [comment]
Hi @fengzhihenxs This usually means the Revocation Endpoint is not supported by your OIDC server. Azure AD doesn't support this.

Grettings Damien

**Comment by fengzhihenxs** [comment]
@damienbod  Do you mean that using Azure AD does not support the revokeRefreshToken and revokeAccessToken methods?

**Comment by damienbod** [comment]
yes, you can see this when you look at the well known endpoints of the Azure AD endpoint

**Comment by rajat-s** [comment]
I too faced this same issue when I set storage to localstorage in OpenIdConfiguration but it works perfectly when I am not setting anything to storage and default is session storage.
Any idea what's going on ? @damienbod
